Snakefile-sorting-hat

```diff
@@ -311,10 +311,13 @@ rule produce_reprocess_call:
         mkdir -p {wildcards.accession}/logs
         exec &> {log}
         pushd {input.working_directory}
-        if ! [ -z {params.lsf_config} ]; then
+        if [ -f {params.lsf_config} ]; then
             if [ ! -f "lsf.yaml" ]; then
                 ln -sf {params.lsf_config} lsf.yaml
```

Comment:
What happens in case `lsf.yaml` does not exist

Reply:
Do you mean the source `{params.lsf_config}` or the local one ?

Reply:
Ignore this...